tests/contenttypes_tests/test_views.py

```diff
@@ -59,7 +92,7 @@ def test_shortcut_with_absolute_url_including_scheme(self):
             response = self.client.get(short_url)
             self.assertRedirects(response, obj.get_absolute_url(), fetch_redirect_response=False)
 
-    def test_shortcut_no_absolute_url(self):
```

Comment:
The current version I have does not contain this file. `test_views.py` was maybe moved to another directory but I can't find it despite my efforts. I tried searching by name (ctrl-p in vim), by content using the fuzzy finder and there's nothing under contenttypes_tests named as `test_views.py`.